### PR TITLE
Add some basic Working with Entities docs.

### PR DESCRIPTION
**Pull request by gabizou**
Adds some basic entities documentation.

### CODE REVIEW

File: source/plugins/entities.rst [hunk 1]
```diff
@@ -2,3 +2,33 @@
 Working with Entities
 =====================
 
+Entities are a huge part of Minecraft, and SpongeAPI in general. All entities exist within worlds
+
+Spawning an Entity
+==================
+
+In almost all cases, to spawn an entity, you need an `EntityType <https://github.com/SpongePowered/SpongeAPI/blob/master/src/main/java/org/spongepowered/api/entity/EntityType.java>`__, a `World <https://github.com/SpongePowered/SpongeAPI/blob/master/src/main/java/org/spongepowered/api/world/World.java>`__, and a `Location <https://github.com/SpongePowered/SpongeAPI/blob/master/src/main/java/org/spongepowered/api/world/Location.java>`__ to do so.
+
```
Comment (simon816) [inline source/plugins/entities.rst]: Should be a link to the online JavaDoc, not the source code.

Comment (gratimax) [inline source/plugins/entities.rst]: we don't have javadocs. Maybe soon.

Comment (gabizou) [inline source/plugins/entities.rst]: We host javadocs?

Comment (simon816) [inline source/plugins/entities.rst]: spongepowered.github.io/SpongeAPI/ There's another page that's using the jdocs too

**Comment by Xemiru** [comment]
Fixes #26.

**Pull request merged**